app/view_models/account_show.rb

```diff
@@ -138,6 +138,10 @@ def generate_backup_codes_partial
     'accounts/actions/generate_backup_codes'
   end
 
+  def backup_codes_generated_at
+    decorated_user.user.backup_code_configurations.order(created_at: :asc).first&.created_at
```

Comment:
Should we add an index for backup_code_configurations.created_at?

Reply:
Probably, good call.

Reply:
Done